src/rqt_graph/dotcode.py

```diff
@@ -634,7 +634,7 @@ def generate_dotgraph(
         rank='same',  # None, same, min, max, source, sink
         ranksep=0.2,  # vertical distance between layers
         rankdir='TB',  # direction of layout (TB top > bottom, LR left > right)
-        simplify=True,  # remove double edges
+        simplify=False,  # do not remove double edges
```

Comment:
TL;DR because otherwise only one edge (i.e. one topic) will be rendered between two nodes.

If `simplify` is `True`, `pydot` stops adding edges to the graph as soon as it sees there's one already (see [here](https://github.com/pydot/pydot/blob/48ba231b36012c5e13611807c9aac7d8ae8c15c4/pydot.py#L1555-L1564)). The true problem here is that [`Edge.__hash__`](https://github.com/pydot/pydot/blob/48ba231b36012c5e13611807c9aac7d8ae8c15c4/pydot.py#L755-L758) does not consider the label.

Reply:
Having said, I guess it should be possible to flip this flag when nodes-only graphs are rendered (and only then). Would you rather do that? I can't picture how we could end up with true duplicates though.

Reply:
I simply don't want existing behavior to be changed / broken.

Reply:
> So you are arguing that it is desired in all cases to show the parallel edges?

My expectation when using `rqt_graph`'s nodes only view would be to see all "connections" between them. Would you agree? Or not? I changed that default instead of selectively enabling it because I don't see how we can end up with parallel edges that convey as much information as a single one. Maybe I'm missing something. @wjwwood thoughts?

Reply:
I just tried a talker publishing on two topics and a listener subscribing to two topic with "Nodes only" and got the parallel edges I wanted to see: 
![rosgraph](https://user-images.githubusercontent.com/1335366/67437453-b49b5b00-f5a5-11e9-86d9-23d91f999433.png)

For the opposite case where a node publishes / subscribes to the same topic multiple time the de-duplication already happens before `dot` so the changed default value doesn't affect it.

Reply:
>  got the parallel edges I wanted to see:

Is that with `simplify=False`?

It seems like that's the desired behavior, I don't exactly follow what the impact is on behavior for this change.

Reply:
The impact of the changed default value is a change in behavior. In this case that the parallel topic edges are both being rendered - which is the desired change.

I was wondering if the opposite is true too: that somewhere we want the simplification to happen but now with the changed default it doesn't happen anymore. Since I couldn't find a case like that while testing the PR I just went ahead an approved it.